### PR TITLE
fix: windows symlink support

### PR DESCRIPTION
**Pull request by ramfox**

### CODE REVIEW

File: .cargo/config [hunk 1]
```diff
@@ -1,2 +1,5 @@
 [alias]
-xtask = "run --package xtask --"
\ No newline at end of file
+xtask = "run --package xtask --"
+
+[target.x86_64-pc-windows-gnu]
+linker = "x86_64-w64-mingw32-gcc"
```
Comment (b5) [inline .cargo/config:5]: can you enlighten me?

**Review by b5 — APPROVED** [review]
this is great

File: .cargo/config (hunk 1 above)
Comment (rklaehn) [reply .cargo/config:5]: This is so you can cross compile for windows on mac and I don't have to go looking for a windows box.

**Comment by b5** [comment]
CI is giving us headaches. This is clearly moving the ball forward on windows. I'm going to merge without waiting for it to finish

**Pull request merged**